tests/lava/lib/dl/slayer/block/test_cuba.py

```diff
@@ -58,27 +74,28 @@ def test_dense_block(self):
         y = net(x)
 
         # export slayer network
-        net.export_hdf5(h5py.File(tempdir + '/cuba_dense.net',
-                                  'w').create_group('layer/0'))
+        net.export_hdf5(
```

Comment:
The changes in this test seems to be linting and just replacing `'` with `"`!

Reply:
Yes, it is just linting. Should I revert it? I just ran over it while constructing the other test ...

Reply:
It is okay, just pointing it out. These create unnecessary spots to review, so its good to try not do it.